content/03.methods.md

```diff
@@ -168,6 +168,20 @@ We annotated putative driver fusions and prioritized fusions based on partners c
 _MYBL1_ [@doi:10.1073/pnas.1300252110], _SNCAIP_ [@doi:10.1038/nature11327], _FOXR2_ [@doi:10.1016/j.cell.2016.01.015], _TTYH1_ [@doi:10.1038/ng.2849], and _TERT_ [@doi:10.1038/ng.3438; @doi:10.1002/gcc.22110; @doi:10.1016/j.canlet.2014.11.057; @doi:10.1007/s11910-017-0722-5] were added to the oncogene list and _BCOR_ [@doi:10.1016/j.cell.2016.01.015] and _QKI_ [@doi:10.1038/ng.3500] were added to the tumor suppressor gene list based on pediatric cancer literature review.
 The fusion filtering workflow can be found in the [OpenPBTA Analysis repository](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/fusion_filtering).
 
+### Mutational Signatures
+
+Using consensus SNVs, we obtained weights for signature sets using deconstructSigs [@doi:10.1186/s13059-016-0893-4 ; @url:https://github.com/raerose01/deconstructSigs] with BSgenome.Hsapiens.UCSC.hg38 annotation [@url:https://bioconductor.org/packages/release/data/annotation/html/BSgenome.Hsapiens.UCSC.hg38.html].
+We used each sample's signature weights to estimate how many mutations for each sample contributed to each signature.
+Weights are normalized from 0 to 1.
```

Comment:
```suggestion
We normalized weights from 0 to 1 with a linear transformation.
```

Reply:
I think this is what you mean. Unless you are saying that the property of the weights is that they are distributed from zero to one.

Reply:
The second thing you said. This is not a transformation I applied but I thing that is already a property of the weights obtained from deconstructSigs. But I felt understanding this helps explain the rationale for the number of contributing mutations estimation.

Reply:
Ok - see if what I wrote seems ok.